src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+interface Movie {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+export const App: FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies: Movie[] = [...moviesFromServer]
```

Comment:
also, there's no need for explicitly setting type here
```suggestion
  const visibleMovies = [...moviesFromServer]
```